lib/rmt/cli/products.rb

```diff
@@ -35,21 +38,87 @@ def list
   end
   map ls: :list
 
-  desc 'enable', 'Enable mirroring of product repositories by product ID or product string.'
+  desc 'enable TARGETS', 'Enable mirroring of product repositories by a list of product IDs or product strings.'
   option :all_modules, type: :boolean, desc: 'Enables all free modules for a product'
-  def enable(target)
-    change_product(target, true, options[:all_modules])
+  long_desc <<-REPOS
+
+Examples:
+
+`rmt-cli products enable SLES/15`
+
+`rmt-cli products enable 1575`
+
+`rmt-cli products enable SLES/15/x86_64,1743`
+
+`rmt-cli products enable --all-modules SLES/15`
+REPOS
+  def enable(*targets)
+    change_products(targets, true, options[:all_modules])
   end
 
-  desc 'disable', 'Disable mirroring of product repositories by product ID or product string.'
-  def disable(target)
-    change_product(target, false, false)
+  desc 'disable TARGETS', 'Disable mirroring of product repositories by a list of product IDs or product strings.'
+  def disable(*targets)
+    change_products(targets, false, false)
   end
 
   protected
 
+  def change_products(targets, set_enabled, all_modules)
+    targets = clean_target_input(targets)
+    raise RMT::CLI::Error.new('No product ids supplied') if targets.empty?
+
+    failed_targets = []
+    targets.each do |target|
+      change_product(target, set_enabled, all_modules)
+    rescue ProductNotFoundException => e
+      puts e.message
+      failed_targets << target
+    end
+
+    unless failed_targets.empty?
+      message = if set_enabled
+                  "Product(s) #{failed_targets.join(',')} could not be found and were not enabled."
+                else
+                  "Product(s) #{failed_targets.join(',')} could not be found and were not disabled."
+                end
+      raise RMT::CLI::Error.new(message)
+    end
+  end
+
   def change_product(target, set_enabled, all_modules)
+    # This will return multiple products if 'SLES/15' was used
+    base_products = find_products(target)
+    raise ProductNotFoundException.new("No product found for target '#{target}'.") if base_products.empty?
+    puts "Found product(s) by target #{target}: #{base_products.map(&:friendly_name).join(', ')}."
```

Comment:
I wonder if we can put some visual separator between each target/product in the output, because for me it looks cluttered and hard to read:

![image](https://user-images.githubusercontent.com/11717425/48547816-c8d1d500-e8cb-11e8-8513-22047c0ac716.png)

Maybe a new line or something?